### PR TITLE
Update Octavia for CLM documentation (SOC-10888)

### PR DESCRIPTION
**Pull request by jeremy-moffitt**
Add command to detail how to find the correct Amphora Image Path
Update tux prompts to Ardana
Add requirement for installation of Octavia client

### CODE REVIEW

**Comment by jeremy-moffitt** [comment]
cherry-pick'd commit from maintenance/cloud_9

**Comment by jeremy-moffitt** [comment]
> I think chris wants to also update and remove all of the neutron lbass-loadbalancer commands with openstack loadbalancer. do we do this in the same PR?

Since that wasnt done in the original change being cherry-picked, I'd rather do that separately if its needed

File: xml/installation-installation-configure_lbaas.xml [hunk 1]
```diff
@@ -160,31 +167,36 @@
   <procedure>
    <step>
     <para>
-     The full path and name for the Amphora image is
-     <filename>/srv/www/suse-12.3/x86_64/repos/Cloud/suse/noarch/openstack-octavia-amphora-image-x86_64-0.1.0-1.21.noarch.rpm</filename>
+     The Amphora image is in an RPM package managed by zypper, display the full path to that file with the following command:
```
Comment (asettle) [inline xml/installation-installation-configure_lbaas.xml]: Could you please wrap this text? It's hard to review :)

File: xml/installation-installation-configure_lbaas.xml [hunk 2]
```diff
@@ -160,31 +167,36 @@
...
+     <screen>&prompt.ardana;find /srv -type f -name "*amphora-image*"</screen>
+     For example:
+     <filename>/srv/www/suse-12.4/x86_64/repos/Cloud/suse/noarch/openstack-octavia-amphora-image-x86_64-0.1.0-13.157.noarch.rpm</filename>
+
+     The Amphora image may be updated via maintenance updates, which will change the filename.
```
Comment (asettle) [inline xml/installation-installation-configure_lbaas.xml]:
```suggestion
     The Amphora image can be updated via maintenance updates, note that this changes the filename.

File: xml/installation-installation-configure_lbaas.xml (hunk 1 above)
Comment (asettle) [inline xml/installation-installation-configure_lbaas.xml]:
```suggestion
     The Amphora image is in an RPM package managed by zypper. Display the full path to the file
     with the following command:

**Pull request merged**